communitybuildingworkinggroup.md

```diff
@@ -0,0 +1,76 @@
+# Community Building Working Group
+
+The Jupyter Community Building Working Group (JCB) acts on behalf of the Project Jupyter
+Executive Council with the objective of growing, building, and connecting the
+Jupyter community of users and contributors. A targeted outcome of this
```

Comment:
I would maybe say that this WG works with the DEI standing committee to improve diversity, equity, and inclusion in our community.

Reply:
We say that below in the more detailed section, so I would repeat that here in summary form.